avionics/common/include/states/mach_lock.h

```diff
@@ -9,20 +9,28 @@ namespace State {
 
 class MachLock : public IState {
   public:
+    /**
+     * @brief Construct a new Mach Lock object
+     * @param next_id
+     * @param MACH_UNLOCK_CHECKS number of mach_checks_ required before next state
+     * @param MACH_UNLOCK_VELOCITY maximum velocity at which mach_checks_ can be done
+     * @param MACH_UNLOCK_TIME maximum amount of time before switching to next state
```

Comment:
Same as above, 'before leaving MACH_LOCK state' or something like that.

Also MACH_UNLOCK_VELOCITY is more a representation of maximum velocity to leave MACH_LOCK rather than that at which the checks can be done